google/fingerprinters/web/scripts/updater/community/spark/versions.txt

```diff
@@ -0,0 +1,18 @@
+v3.1.3
+v3.2.1
+v3.2.2
+v3.2.3
+v3.2.4
+v3.3.0
+3.3.1
+v3.3.2
+3.3.3
+3.4.0
+3.4.1
+3.4.2
+3.4.3
+3.4.4
+3.5.0
+3.5.1
+3.5.2
+3.5.3
```

Comment:
```suggestion
3.5.3
3.5.4
```

Hey @VickyTheViking, I just noticed there's a new version 3.5.4. Please apply the suggestion and I'll mark the contribution as good to go.